war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -242,6 +249,10 @@ var FormChecker = {
     }
 }
 
+if (performance.getEntriesByType("resource")[0].nextHopProtocol === "h2") {
```

Comment:
Is this defensive enough for this API? E.g., what if the list is empty? Can that happen?

Reply:
extracted the check to a method and safe guarded the array access and the field access